### ISSUE TITLE
Update to `config_split:2.0.x` and implement config changes

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#4137]
# Motivation

Follow-up to #4035.

# Proposed solution
- [x] Update `config_split` to 2.0.x version.
- [x] Convert `config_split.config_split.*.yml` files to new format.
- [x] Apply necessary patches.
- [x] #4237 
- [x] #4316
- [x] #4159
- [x] #4401
- [x] Convert feature splits ([instructions](https://github.com/uiowa/uiowa/pull/4091#issuecomment-919514590)):
  - [x] apr
  - [x] areas_of_study
  - [x] event
  - [x] grad_person_extended
  - [x] profiles
  - [x] scholarship
  - [x] sitenow_v2
  - [x] thesis_defense
  - [x] topic_page
- [x] Convert site splits:
  - [x] admissions.uiowa.edu
  - [x] brand.uiowa.edu
  - [x] coronavirus.uiowa.edu
  - [x] education.uiowa.edu
  - [x] genetics.grad.uiowa.edu
  - [x] grad.uiowa.edu
  - [x] hr.uiowa.edu
  - [x] icsa.uiowa.edu
  - [x] ighn.international.uiowa.edu
  - [x] immuno.grad.uiowa.edu
  - [x] iowasuperfund.uiowa.edu
  - [x] pharmacy.uiowa.edu
  - [x] policy.clas.uiowa.edu
  - [x] presidentialsearch.uiowa.edu
  - [x] sitenow.uiowa.edu
  - [x] sppa.uiowa.edu
  - [x] tippie.uiowa.edu
  - [x] transportation.uiowa.edu
  - [x] uiowa.edu
- [x] Account for permission changes in https://github.com/uiowa/uiowa/pull/4139/files

**Comment by pyrello** [comment]
This is blocked by #4160 and #4159

**Comment by pyrello** [comment]
I would consider #4160 to be a "nice to have" instead of necessary for this moving forward.

**Comment by pyrello** [comment]
As I started working on re-exporting the admissions site split, I encountered the following issue: https://www.drupal.org/project/config_split/issues/3240272#comment-14258004
